.circleci/config.yml

```diff
@@ -250,7 +250,9 @@ jobs:
       - store_artifacts:
           path: ./integration/forks/logs
   build-publish-explorer:
-    machine: true
+    machine: 
+        image: circleci/classic:latest
```

Comment:
Is there a tag we could use - hate for our builds to fail because a new image was published that broke.

Reply:
`201808-01` [looks like the latest](https://circleci.com/docs/2.0/configuration-reference/#available-machine-images). Just swapped it in.

Reply:
@j16r @se3000 FYI if you specify `machine: true`, circleci will use the latest by default.